Resources/migration.yml

```diff
@@ -329,3 +329,6 @@ WeaponSubMachineGunDrozdRubber: WeaponSubMachineGunDrozd
 WeaponRifleLecterRubber: WeaponRifleLecter
 ReinforcementRadioSyndicateMonkey: ReinforcementRadioSyndicateAncestor
 ReinforcementRadioSyndicateMonkeyNukeops: ReinforcementRadioSyndicateAncestorNukeops
+
+# 2024-04-27
+ClosetEmergencyN2FilledRandom: ClosetImprovisedEmergencyN2FilledRandom
```

Comment:
This will make it impossible to map the `ClosetEmergencyN2FilledRandom` closets. They will always be changed to the "improvised" version.

Reply:
Thank you for raising this, I'll likely be dropping the migration in favour of the solution above.